nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -118,6 +118,8 @@ In addition to numerous new and updated packages, this release has the following
 
 - [nimdow](https://github.com/avahe-kellenberger/nimdow), a window manager written in Nim, inspired by dwm. Available as [services.xserver.windowManager.nimdow.enable](options.html#opt-services.xserver.windowManager.nimdow.enable).
 
+- [onedriver](https://github.com/jstaf/onedriver), a network filesystem that gives your computer direct access to your files on Microsoft OneDrive.
```

Comment:
This should go into the 23.11 release notes now.

Reply:
Ok, I will create a new MR based on the master branch then, it will be easier than rebasing :)

Reply:
Created a new MR with the modification in the right file #259753